src/sections/Blog/Related-Posts/index.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
```

Comment:
Is this a new file or so?

Reply:
I think this file is no more needed, as the latest code changes have moved this part to a separate component. Please confirm once.

Reply:
Yeah. It's not needed anymore

Reply:
So @Tanuj22 & @Nikhil-Ladha Should I remove this file?

Reply:
Yes, please. Actually, it's not there in the current master code. So, you can remove it in this PR as well

Reply:
The whole file or the whole directory itself?

Reply:
The complete `Related-posts` directory.